data/sql/updates/pending_db_world/rev_1617549311457174700.sql

```diff
@@ -0,0 +1,31 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1617549311457174700');
+
+ALTER TABLE `spell_dbc`
+    CHANGE `Description_Lang_enUS` `Description_Lang_enUS` text(0) DEFAULT NULL,
+    CHANGE `Description_Lang_enGB` `Description_Lang_enGB` text(0) DEFAULT NULL,
+    CHANGE `Description_Lang_koKR` `Description_Lang_koKR` text(0) DEFAULT NULL,
+    CHANGE `Description_Lang_frFR` `Description_Lang_frFR` text(0) DEFAULT NULL,
+    CHANGE `Description_Lang_deDE` `Description_Lang_deDE` text(0) DEFAULT NULL,
+    CHANGE `Description_Lang_enCN` `Description_Lang_enCN` text(0) DEFAULT NULL,
+    CHANGE `Description_Lang_zhCN` `Description_Lang_zhCN` text(0) DEFAULT NULL,
+    CHANGE `Description_Lang_enTW` `Description_Lang_enTW` text(0) DEFAULT NULL,
+    CHANGE `Description_Lang_zhTW` `Description_Lang_zhTW` text(0) DEFAULT NULL,
+    CHANGE `Description_Lang_esES` `Description_Lang_esES` text(0) DEFAULT NULL,
+    CHANGE `Description_Lang_esMX` `Description_Lang_esMX` text(0) DEFAULT NULL,
+    CHANGE `Description_Lang_ruRU` `Description_Lang_ruRU` text(0) DEFAULT NULL,
+    CHANGE `Description_Lang_ptPT` `Description_Lang_ptPT` text(0) DEFAULT NULL,
+    CHANGE `Description_Lang_ptBR` `Description_Lang_ptBR` text(0) DEFAULT NULL,
+    CHANGE `Description_Lang_itIT` `Description_Lang_itIT` text(0) DEFAULT NULL,
+    CHANGE `Description_Lang_Unk` `Description_Lang_Unk` text(0) DEFAULT NULL;
```

Comment:
text(0)?
so no limits?

Reply:
yes, there are description too long in game for spells

Reply:
I am not sure that the spell_dbc description fields affect the description that you see in the client, but that's ok

Reply:
for example, this warrior description for execute was like this:
`Attempt to finish off a wounded foe, causing ${$m1+$AP*0.2} damage and converting each extra point of rage into $*10;F1 additional damage (up to a maximum cost of 30 rage).  Only usable on enemies tha`
instead
like this:
`Attempt to finish off a wounded foe, causing ${$m1+$AP*0.2} damage and converting each extra point of rage into $*10;F1 additional damage (up to a maximum cost of 30 rage).  Only usable on enemies that have less than 20% health.`

Reply:
ok :+1:

Reply:
> I am not sure that spell_dbc description fields affect the description that you see in the client, but that's ok

Yeah i confirm, it doesn't affect the client description, but if a day we will find out a way to do it, we will have a lot of bullshit wrote there, an then we will have fix the texts as well.. i would like avoid it..